Program.cs

```diff
@@ -1,5 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
 namespace Delegates
 {
+    public static class ArrayExtensions
```

Comment:
An excellent method, but it already exists in System.Collections.Generic

Reply:
not in the older versions of vs like 2019 and before